package/Dockerfile

```diff
@@ -19,8 +19,14 @@ COPY ./main.go ./main.go
 COPY --from=xx / /
 
 ARG TARGETPLATFORM
+ARG COMMIT
+ARG VERSION
 ENV CGO_ENABLED=0
-RUN xx-go build -o /aks-operator && xx-verify /aks-operator
+RUN xx-go build -ldflags \
+           "-X github.com/rancher/aks-operator/pkg/version.GitCommit=${COMMIT} \
```

Comment:
Don't we need to include a `version` package with `GitCommit` and `Version` variables for this to work? From a quick look, I didn't find such a package.

Reply:
In case the build was happening with Go having access to the `.git` dir, then `go build` would add the metadata info directly in `vcs.revision` and other related fields. Given that we are building inside the container and only copying some of the Go related files, `go build` cannot extract those metadata. That's why I proposed to add them.

Reply:
Ok cool. I was thinking about doing this exact thing for https://github.com/rancher/aks-operator/issues/726 but I will make a follow-up PR once this PR gets merged.

Reply:
Nice 👍🏻